### DISCUSSION TITLE
Any plans on Go driver?

### DISCUSSION THREAD

**Discussion by yznts** [duckdb/duckdb#3807] [General]
Hey guys!
First of all, nice work! I really like how fast became my realtime analytics queries.
Anyway, I'm still suffering with own duckdb CLI wrapper (with startup delays on each query) instead of normal driver, because my project uses Go.
I'm not experienced enough to write a driver on my own, that's why I'm asking here, maybe there are some plans on this direction?

**Comment by hannes** [comment]
Have a look here: https://github.com/marcboeker/go-duckdb

**Reply by delaneyj** [reply]
This driver is using `database/sql` interface.  For an embedded DB you really want similar to [ZombieZen's approach](https://github.com/zombiezen/go-sqlite).  Further reasoning available [here](https://crawshaw.io/blog/one-process-programming-notes).
